home/views.py

```diff
@@ -7,3 +9,17 @@ def about(request):
 
 def landingpage(request):
     return render(request, 'home/landingpage.html')
+
+
+def new_question(request):
+    form = QuestionForm
+    if request.method == 'POST':
+        questForm = QuestionForm(request.POST)
+        if questForm.is_valid():
+            questForm = questForm.save(commit=False)
+            for currProfile in Profile.objects.all():
+                if (currProfile.user == request.user):
+                    questForm.profile = currProfile
+                    break
+            questForm.save()
+    return render(request, 'home/questions/new_question.html', {'form': form})
```

Comment:
you can add here next to 'form':form another variable - 'title': '<title-name>'
in order to edit the title of the new question page that will be shown on top of the tab in the browser

Reply:
Ok, I added 'title': 'New Question'